SWLOR.Game.Server/Feature/PerkDefinition/DevicesPerkDefinition.cs

```diff
@@ -297,21 +297,21 @@ private void IncendiaryBomb()
                 .Name("Incendiary Bomb")
 
                 .AddPerkLevel()
-                .Description("Creates a fire field at the explosion site, dealing 4 fire DMG to all creatures who enter the area of effect for 20 seconds. Consumes explosives on use.")
+                .Description("Creates a fire field at the explosion site, dealing 4 fire DMG, scaling with your Percepiton Score, to all creatures who enter the area of effect for 20 seconds. Consumes explosives on use.")
```

Comment:
There's a bunch more typos below. Copy/pasting can be bad ;) Please fix them all